### DISCUSSION TITLE
Can I sort by most recently accessed?

### DISCUSSION THREAD

**Discussion by ethanuppal** [ibhagwan/fzf-lua#1454] [Q&A] [answered]
Is there a way to sort the results of `files` and `grep` in the order that the (corresponding) files were last accessed (i.e., opened in an editor pane)?

**Comment by ibhagwan** [comment, marked as answer]
The lists are the output of the underlying command `fd` for files and `rg` for grep (if installed), you can send custom flags using `fd_opts` and `rg_opts` and use `--sort=...` but I'm not aware of any that sort by last accessed.

If you want to sort by last accessed use `:FzfLua oldfiles`.

**Reply by ethanuppal** [reply]
I was looking for something similar to VSCode's `<Cmd-P>` (on macOS). This works for now though, thanks!
